scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -183,8 +191,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.figure(figsize = (13, 13))\n",
-    "plt.imshow(plt.imread(os.path.join(segmentation_dir, \"Point8_HH3_Membrane_overlay.tiff\")))"
+    "# TODO: Viz for new deepcell format\n",
```

Comment:
Here's what this could look like. Either provide a list of point names, or if an empty list assume all points. 
It will then load the corresponding deepcell_input points and the segmentation in the output folder. Those two things will get fed into plot_utils.plot_overlay, and saved into the same folder with the deepcell_output segmentations.